### ISSUE TITLE
Tidy Makefile

### ISSUE THREAD

**Issue by John-P** [TIA-Lab/tiatoolbox#8]
Looking through the Makefile we probably want to tidy it up a bit. For example,. there is a lint option with flake8 but flake8 is not in the requirements files. Also, we probably want to add a format action to format the code with black. Lastly, the coverage action is set to use the python coverage module whereas we are currently using pytest-cov (although maybe we should try coverage).

Related to #6, #7 and #9.

In summary, issues to resolve:

- [ ] Change code coverage to pytest-cov OR swap to using coverage instead of pytest-cov on travis (See #7)
- [x] Add flake8 to requirements for linting? (See #9) ~~OR remove the lint action from the make file.~~
- [ ] Add format action to run [black](https://github.com/psf/black) on the code? See #9.
- [ ] Do we even need this Makefile?
- [ ] Check the other actions in the Makefile.

**Comment by shaneahmed** [comment]
Can we add pre-commit hooks for black and flake8?
https://black.readthedocs.io/en/stable/version_control_integration.html
https://ljvmiranda921.github.io/notebook/2018/06/21/precommits-using-black-and-flake8/

**Comment by shaneahmed** [comment]
I think this issue can be closed now.

**Comment by shaneahmed** [comment]
Issue resolved.
